Sources/DangerShellExecutor/ShellExecutor.swift

```diff
@@ -82,8 +106,17 @@ public struct ShellExecutor: ShellExecuting {
 
     private func makeTask(for command: String,
                           with arguments: [String],
-                          environmentVariables: [String: String]) -> Process {
-        let script = "\(command) \(arguments.joined(separator: " "))"
+                          environmentVariables: [String: String],
+                          outputFile: String?) -> Process {
+        let scriptOutputFile: String
```

Comment:
var scriptOutputFile = "" and drop the else statement?

Reply:
I always avoid the `var` usage where I can, I prefer to keep the variables immutable where possile